docs/repositories.md

```diff
@@ -511,6 +511,17 @@ You can prevent this by adding double dashes to prevent any following argument f
 poetry config -- http-basic.pypi myUsername -myPasswordStartingWithDash
 ```
 
+{{% note %}}
+In some cases like that of [Gemfury](https://gemfury.com/help/errors/repo-url-password/) repositories, it might be
+required to set an empty password. This is supported by Poetry.
+
+```bash
+poetry config http-basic.foo <TOKEN> ""
+```
+
+However, it is important to note that empty usernames are not supported. And attempts use it will fail unpredictably.
```

Comment:
```suggestion
However, it is important to note that empty usernames are not supported - and attempts to use it will fail unpredictably.
```

Maybe?

Reply:
Good catch. I originally meant to write "Any attempts". Rewrote it to be more direct.